### PR TITLE
[EYE-81] Bluetooth and Camera Registration Functionality in the Mobile App

### PR DESCRIPTION
**Pull request by matthew-latanafrancia**
# Description
This PR includes changes that will allow the user to connect to the pi via bluetooth.  After a user connects to the pi, the user will be able to add a user and password depending on the type of connection that the wifi it currently is onn

Better looking UI will be touchup in a different PR which includes

## Testing

Local testing: Locally, going from logging in < Group < + on Camera screen, you can start the camera registration.  I tested that this was working and that all bluetooth services were working.

**Screenshots of testing is to be added**

Unit testing: 

## JIRA Ticket Link
**To be added**

### CODE REVIEW

**Review by 1Revenger1 — APPROVED** [review]

File: app/human-detector-app/src/ble/bleServices.ts [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+    this.connectedDevice = await deviceConnection.discoverAllServicesAndCharacteristics();
+    this.bleManager.stopDeviceScan();
+  };
+
+  /**
+   * Disconnect from current device
+   */
+  public async disconnectFromDevice() {
+    if (this.connectedDevice) {
+      await this.connectedDevice.cancelConnection();
```
Comment (matthew-latanafrancia) [inline app/human-detector-app/src/ble/bleServices.ts:109]: Why do we need an await here?

Comment (1Revenger1) [reply app/human-detector-app/src/ble/bleServices.ts:109]: Didn't want the device to become null until all transactions and stuff were ended, just to be safe.

File: app/human-detector-app/.eslintrc.json [hunk 2]
```diff
@@ -22,5 +22,6 @@
         "react-native"
     ],
     "rules": {
+        "no-console": "off"
```
Comment (tucker-moore) [inline app/human-detector-app/.eslintrc.json]: Thank you, the bazillion warnings on every PR were getting annoying

**Review by tucker-moore — APPROVED** [review]

**Review by matthew-latanafrancia — COMMENTED** [review]
can't approve because im the pr author but lgtm thanks for waiting :)

**Pull request merged**